virttest/qemu_monitor.py

```diff
@@ -1004,8 +1004,9 @@ def get_backingfile(self, device):
         """
         backing_file = None
         block_info = self.query("block")
+        block_info = block_info.replace('\n', ' ')
```

Comment:
Why replace '\n' to empty strings here, if you want to use '.' match newline, use 're.S | re.M' please.